Xamarin.Forms.Core/IBorderElement.cs

```diff
@@ -1,11 +1,21 @@
-﻿namespace Xamarin.Forms
+﻿using System.ComponentModel;
+
+namespace Xamarin.Forms
 {
-	interface IBorderElement
+	[EditorBrowsable(EditorBrowsableState.Never)]
+	public interface IBorderElement
```

Comment:
I made this public because the android/ios platform needs access to these properties

Reply:
yeah, I guess it makes the renderers code simpler as well